src/script/auth/module/action/AuthAction.ts

```diff
@@ -152,6 +152,13 @@ export class AuthAction {
         await apiClient.api.user.postVerificationCode(email, VerificationActionType.LOGIN);
         dispatch(AuthActionCreator.successfulSendTwoFactorCode());
       } catch (error) {
+        if (error.label === BackendError.LABEL.BAD_REQUEST) {
+          error = new BackendError({
+            code: StatusCodes.BAD_REQUEST,
+            label: BackendError.AUTH_ERRORS.EMAIL_REQUIRED,
```

Comment:
Is `"email required"` the only case `BAD_REQUEST` error could be thrown? There's probably no way to differentiate it, right?

Reply:
It is the only case for this endpoint yes